conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -119,3 +121,9 @@ def cleanup_if_exists(self, conjurrc_conjur_url: str):
                     f"Cleanup failed for key '{attr}' from the '"
                     f"{KeystoreWrapper.get_keyring_name()}' "
                     f"credential store.\n{traceback.format_exc()}")
+
+    def get_store_location(self):
+        """
+        Method to return the source of the credentials
+        """
+        return f'{KeystoreWrapper.get_keyring_name()} credential store'
```

Comment:
KeystoreWrapper.get_keyring_name() - Call it once and populate a private field for reuse.